users/models.py

```diff
@@ -120,6 +121,11 @@ def get_full_name(self):
         return self.name
 
 
+def __str__(self):
```

Comment:
Looks like this was supposed to be a member of `UserManager` (i.e.: should be indented)